spec/parameterizer/parameterizer.spec.ts

```diff
@@ -1,31 +1,318 @@
 import * as ganache from 'ganache-cli'
 import Web3 from 'web3'
+import { Contract } from 'web3/types'
 import Parameterizer from '../../src/contracts/parameterizer'
-import { maybeParseInt } from '../../src/helpers'
+import { PARAMETERIZER_ABI, ONE_ETHER, ONE_GWEI } from  '../../src/constants'
+import { Return } from '../../src/@types'
+import {
+  deploy,
+  readBytecode,
+  call,
+  transact,
+} from '../../src/helpers'
 
 const provider:any = ganache.provider(),
-  w3 = new Web3(provider)
+  w3 = new Web3(provider, undefined, {defaultBlock: 'latest',
+    transactionConfirmationBlocks: 1, transactionBlockTimeout: 5}),
+    toBN = w3.utils.toBN
 
-let accounts:string[],
-  parameterizer:Parameterizer
+let parameterizer:Parameterizer,
+  accounts:string[],
+  deployed:Contract
 
 describe('Parameterizer', () => {
-  beforeAll(() => {
+  beforeAll(async () => {
+    accounts = await w3.eth.getAccounts()
 
-  })
+    // deploy it...
+    const bin:string = readBytecode('voting')
 
-  afterAll(() => {
+    deployed = await deploy(w3,
+                            accounts[0],
+                            PARAMETERIZER_ABI,
+                            bin,
+                            [accounts[0],
+                              10000,
```

Comment:
let's put some real numbers in here for posterity.

you should be able to use the same initial values i used the .py conftest setup for p11r.

there are already `const`s for all the Eth units, and the same `web3.toWei` helpers exist in web3.js that exist in web3.py

Reply:
done